flows/vision-zero/db_permission_grantor/db_permission_grantor.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+import os
+import json
+
+import prefect
+from prefect.backend import get_key_value
+from prefect import Flow, task
+from prefect.run_configs import UniversalRun
+import psycopg2
+
+kv_store = get_key_value("Vision Zero")
+kv_dictionary = json.loads(kv_store)
+
+DB_HOST = None
+DB_USER = None
+DB_PASS = None
+DB_NAME = None
+SCHEMAS_CDL = None
+STAFF_CDL = None
+
+if False:
```

Comment:
I have vague memory of this pattern from before, but can't remember why we use `if False`. is it for local development?

Reply:
It's an ugly pattern, and it's entirely from my hands. 🙋🏼‍♂️

I really want to write a function which detects if it's running in 'agent' mode or if it's being run via an invocation of `.run()` in the script, and based on that pulls variables from the KV store or the actual environment.

I've lamented not knowing how to detect running in either state, so i've just been propagating this ugly boolean to switch between the development/production configurations. I also have not put much of an attempt to make that distinction - it would be a neat micro-skill-share in a prefect meeting discovery maybe.

Reply:
I did a quick search in the prefect python module documentation, and I didn't see a method that was jumping out at me that would indicate if the flow is running via a schedule & agent vs. being invoked on the command line. If you think it's valuable, we can spin off an issue to work out this development vs. production configuration mechanism.

Reply:
Maybe you could do something like [this](https://github.com/cityofaustin/atd-prefect/blob/74eec77e8cb04ce6ce8015e39304014708e82e77/flows/test/skillshare.py#L17) where you have a env variable defined inside your dockerized agent that would tell you if you are running locally?